setup.cfg

```diff
@@ -19,24 +19,20 @@ classifiers =
 [options]
 packages = find:
 install_requires =
+    PyCifRW~=4.4
     aiida-core>=2.0,<3
-    aiidalab>=21.11.2
     aiidalab-eln>=0.1.2,~=0.1
     ansi2html~=1.6
-    ase<3.20
+    ase~=3.18
     bokeh~=2.0
     humanfriendly~=10.0
     ipytree~=0.2
-    ipywidgets~=7.6
-    jupyter-client<7
+    ipywidgets~=7.7
     more-itertools~=8.0
-    nbconvert<6
     nglview~=3.0
-    numpy~=1.17
     optimade-client==2022.9.19
-    pandas~=1.0
```

Comment:
@unkcpz I just noticed that you removed pandas here. Do you know where else it is coming from?

Reply:
Ah, it seems it comes from `optimade-client`, never mind.